content/sensu-go/5.17/guides/secrets-management-vault.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Secrets management with HashiCorp Vault"
+linkTitle: "Secrets Management with Vault"
+description: "Sensu's secrets management allows you to avoid exposing secrets in your Sensu configuration. In this guide, you'll learn how to use HashiCorp Vault as your external secrets management provider and refer to external secrets in your Sensu configuration."
+weight: 175
+version: "5.17"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.17:
+    parent: guides
+---
+
+- [Create your secrets provider](#create-your-secrets-provider)
+- [Create your secrets](#create-your-secrets)
+- [Add a check, handler, and mutator](#add-a-check-handler-and-mutator)
+- [**NEXT STEP NEEDED**](#next-step-needed)
+- [Next steps](#next-steps)
+
+Sensu's secrets management allows you to avoid exposing secrets in your Sensu configuration.
+In this guide, you'll learn how to use [HashiCorp Vault][1] as your external [secrets provider][2] and authenticate via the HashiCorp Vault integration's [token auth method][3] or [TLS certificate auth method][4].
+
+To follow this guide, you’ll need to [install the Sensu backend][5], have at least one [Sensu agent][11] running on Linux, and [install and configure sensuctl][7].
+
+_**NOTE**: Secrets can only be passed to checks if you have configured mTLS for the agent and backend._
+You will also need to have [HashiCorp Vault][15] set up.
```

Comment:
Is it possible to add a minimum viable Vault setup to make it easier for people to follow this guide and see for themselves that it works?